### ISSUE TITLE
Documentation detail

### ISSUE THREAD

**Issue by WGroleau** [Sh1d0w/clean-links#16]
This is not a bug in the plug-in, but it might be worth adding to your documentation that fbclid (and probably the others) causes a 404 (not found) if the link is to something on a WordPress site.

**Comment by Sh1d0w** [comment]
That's interesting can you provide example link so I can reproduce, please?

**Comment by WGroleau** [comment]
Well, I am puzzled.  I got 404 many times when I clicked links to my blog in Facebook.  I made it stop by putting in my .htaccess this:
```
<IfModule mod_rewrite.c>
RewriteEngine On
  RewriteCond %{QUERY_STRING} ^(.*?)(&?(fbclid|gclid|utm_[a-zA-Z0-9_-]+)=[a-zA-Z0-9_-]+)$
  RewriteRule ^(.*)$ /$1?%1
  # [L,NE,R=301]
</IfModule>

```I removed that for the sake of demonstrating, but it doesn't get 404 now!

**Comment by Sh1d0w** [comment]
Ah I see, misconfiguration on your end.

**Comment by Sh1d0w** [comment]
If it happens again please copy the url from FB and the url you are redirected after clicking on it, so I can investigate.

**Comment by WGroleau** [comment]
On Jun 29, 2021, at 11:55, Sh1d0w ***@***.***> wrote:
> Ah I see, misconfiguration on your end.

I don’t know.  The 404s happened for months on links from Facebook.  
They stopped when I made that change in .htaccess
I reverted the .htaccess to EXACTLY what it was before, but the 404s did not come back.

— 
Wes Groleau
伟思礼

  I have discovered in my 60+ years of ministry
  that I have won more to Christ by just being present
  when they need someone to display the love of God.
  I didn't have to preach or quote a scripture.
  Somehow God takes over from there.
            J. R. Farley
